snmp/datadog_checks/snmp/data/default_profiles/f5-big-ip.yaml

```diff
@@ -256,6 +256,40 @@ metrics:
           OID: 1.3.6.1.4.1.3375.2.2.10.1.2.1.1
           name: ltmVirtualServName
 
+  - MIB: F5-BIGIP-LOCAL-MIB
+    table:
+      OID: 1.3.6.1.4.1.3375.2.2.10.13.2
+      name: ltmVsStatusTable
+    symbols:
+      # Gauges
+      - name: ltmVsStatus
+        constant_value_one: true
+    metric_tags:
+      - tag: ltm_vs_status_avail_state
+        column:
+          OID: 1.3.6.1.4.1.3375.2.2.10.13.2.1.2
+          name: ltmVsStatusAvailState
+        mapping:
+          0: error
+          1: available in some capacity
+          2: not currently available
+          3: not available
+          4: availability is unknown
+          5: unlicensed
```

Comment:
Same as this comment: https://github.com/DataDog/integrations-core/pull/15049/files#r1247638723

We should use original values when possible for consistency (underscore case, no space):

```
ltmVsStatusAvailState OBJECT-TYPE 
	SYNTAX INTEGER {
		none(0),
		green(1),
		yellow(2),
		red(3),
		blue(4),
		gray(5)
	}
```